combat-trainer.lic

```diff
@@ -2507,7 +2508,7 @@ class TrainerProcess
       waitrt?
       fix_standing
     when /^Tactics$/i
-      bput($tactics_actions.sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty?
+      bput($tactics_actions.sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty? || !game_state.is_engaging_allowed?
```

Comment:
I'm not super familiar with Innocence (I just read its entry on elanthipedia), so maybe this is a dumb question. Why would someone using Innocence also be training Tactics?

Reply:
It's fair, there are a couple "should never happen" examples here -- see also training Astrology, since Innocence is an empath signature spell. However, it also felt like `is_engaging_allowed?` could potentially be broader than whether or not you have Innocence cast, so I tried to cover pretty much any case that would lead to a `face` or `engage` command being issued. If that doesn't make sense I could also see this being restricted to only cases that would actually happen for an empath using the spell.